ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ComparisonUtil.java

```diff
@@ -38,25 +38,31 @@
   private ComparisonUtil() {
   }
 
-  static void isValidComparison(
+  public static void assertValidComparison(
```

Comment:
This method appears to only be called from tests.... so should probably be deleted, and the tests changed to check the return value of `isValidComparison`